### ISSUE TITLE
[Feature Request] WinApps Application Icons Modification

### ISSUE THREAD

**Issue by KernelGhost** [winapps-org/winapps#140]
In order to differentiate between duplicate applications installed on the host GNU+Linux system and those installed on Windows, it might be a good idea to modify the icon files of the officially supported applications to feature a small WinApps icon in the bottom left of the icon, similar to how shortcuts are displayed visually in Windows.

I have prepared this SVG file of the WinApps logo, if anyone would like to experiment with this idea.

![AppIcon](https://github.com/user-attachments/assets/13ffaa80-cb1c-441c-9f5a-ce7a5f9ee6df)

**Comment by LDprg** [comment]
@oskardotglobal what do you think about this? I would however prefer if the original icon is preserved and a separate script adds the logo to a copy. This way we could easily change the individual and winapps logos.

**Comment by KernelGhost** [comment]
I agree that the original icons should be preserved. Although I'm not familiar with automating the modification of vector graphics, it should be possible to implement.

**Comment by KernelGhost** [comment]
Here's an example to illustrate what I mean.
![Example](https://github.com/user-attachments/assets/8260c5fe-de5e-49db-b85a-328d2b0a1ac5)
I'm by no means a graphic designer, so please feel free to share other concepts or ideas.
